Flask-Integration/get-localhost.py

```diff
@@ -1,19 +1,27 @@
 from flask import Flask, jsonify, request
+from flask_cors import CORS, cross_origin
 from spellCheck import *
+import json
 
 #export FLASK_APP=get-localhost.py
 #python -m flask run
 
 app = Flask(__name__)
 
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
```

Comment:
This is a little difficult to understand, we'd just get together once to understand this better.

Reply:
Sure, although I am not entirely familiar on what it does, I know CORS is a special permission that the API uses as a protocol to be accessible from the browser, we can do some research about it